105-server-side-apply-implementation-fg-timelines.md

```diff
@@ -0,0 +1,75 @@
+# Server Side Apply - Implementation Split, Feature Gates, and Timelines
+
+This proposal builds on the original [Server Side Apply proposal](052-k8s-server-side-apply.md), which outlines the technical specification and behavior of Kubernetes Server Side Apply (SSA).
+
+Server Side Apply enables declarative updates to Kubernetes resources by tracking ownership of individual fields through the `.metadata.managedFields` property. 
+This mechanism allows multiple controllers to safely modify different parts of the same resource without overwriting each other’s changes.
+
+With Server Side Apply:
+- Strimzi will explicitly claim ownership only of the fields it manages.
+- Other tools can safely annotate or patch resources without triggering reconciliation loops.
+- Reconciliation is simplified because Strimzi no longer needs to diff the entire resource or filter changes manually.
+
+Strimzi will apply the full desired state of each resource during reconciliation and apply it using Server Side Apply, allowing the Kubernetes API server to handle merging and field ownership.
+This eliminates the need for comparing actual and desired states, as well as related `GET` calls or the current `ResourceDiff` logic.
+
+To reduce the risk of conflict, Server Side Apply operations will initially use `force = false`, retrying with `force = true` only when necessary. 
```

Comment:
Wasn't this more to be able to log the conflict? Because if you force it anyway afterwards, you do not really reduce any risk.